src/Eloquent/SpatialTrait.php

```diff
@@ -134,7 +134,7 @@ public function scopeDistance($query, $geometryColumn, $geometry, $distance)
     {
         $this->isColumnAllowed($geometryColumn);
 
-        $query->whereRaw("st_distance(`$geometryColumn`, ST_GeomFromText(?, ?, 'axis-order=long-lat')) <= ?", [
+        $query->whereRaw("st_distance(`$geometryColumn`, ST_GeomFromText(?, ?)) <= ?", [
```

Comment:
I think this is for MySQL 5 compatibility, see https://dev.mysql.com/blog-archive/axis-order-in-spatial-reference-systems/